modules/ROOT/pages/product-versioning.adoc

```diff
@@ -26,6 +26,16 @@ Here is a Bonita Platform version / Technical Id / UI Designer version correspon
 |===
 | Release date | Bonita Platform version | Technical Id | UI Designer version
 
+| 2022-03-07
+| 2021.1-0307
+| 7.12.11
+| 1.12.?
+
+| 2022-02-02
+| 2021.1-0202
+| 7.12.10
+| 1.12.?
```

Comment:
❓ Can someone provide the UID version please?

Reply:
:heavy_check_mark:

Reply:
thanks